utils/hostapi.d.ts

```diff
@@ -81,6 +86,18 @@ export interface AzureHostExtensionApi {
     //#endregion
 }
 
+export interface PickAppResourceOptions extends IAzureQuickPickOptions {
+    /**
+     * App resource type to filter the app resource picks
+     */
+    type?: string;

```

Comment:
I think we should go ahead and add "kind" to the options as well.

Reply:
Updated it to have a filter property with both type and kind, with kind optional.